arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -877,6 +878,24 @@
 		F4E5CD7E207058D8000A6D35 /* FeatureLayerRenderingModeMapViewController.swift in CopyFiles */ = {isa = PBXBuildFile; fileRef = F4E5CD7C20704AAA000A6D35 /* FeatureLayerRenderingModeMapViewController.swift */; };
 /* End PBXBuildFile section */
 
+/* Begin PBXBuildRule section */
+		007A424F256C747E0025F582 /* PBXBuildRule */ = {
+			isa = PBXBuildRule;
+			compilerSpec = com.apple.compilers.proxy.script;
+			filePatterns = "*.masque";
+			fileType = pattern.proxy;
+			inputFiles = (
+			);
+			isEditable = 1;
+			name = "Generate swift code from secrets";
+			outputFiles = (
+				"$(DERIVED_FILE_DIR)/$(INPUT_FILE_BASE)",
+			);
+			runOncePerArchitecture = 0;
+			script = "rm -f \"${SCRIPT_OUTPUT_FILE_0}\"\n\"${SRCROOT}/Scripts/masquerade\" -i \"${INPUT_FILE_PATH}\" -o \"${SCRIPT_OUTPUT_FILE_0}\" -s \"$SRCROOT/.secrets\"\n";
```

Comment:
During the verification Sarat spotted this:

> If we build with an empty `.secrets` file in the first run, and then add the keys into it, the `AppSecrets.swift` file is not overwritten - it still has empty keys and make the sample crash. Clean build folder and rebuild fix that.

In the `masquerade` script there is a `-f` flag.  Without setting it (or FORCE_WRITE='false'), the script stops running if the output file exists. But when it is set to `FORCE_WRITE='true'`, the script doesn't actually force overwrite the output file.

I tried to add -f flag on masquerade line 146 to `mv -f "${TEMP_FILE}" "${OUTPUT_FILE}"` or add `rm -f "${OUTPUT_FILE}"` without success. I don't understand why... 🤯 

So instead add another line in the build rule to remove the output file in the first place, to avoid this issue.
Would like to know your thoughts!